packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -354,6 +369,23 @@ export default function Combobox({
         ? uniqueId
         : pillIdPrefix;
 
+    /**
+     * Filters the items based on the input value. This is used when the
+     * `autoComplete` prop is set to `list` or `both`.
+     */
+    const filterItems = React.useCallback(
+        (value: string) => {
+            return children.filter((item) => {
+                const label = item.props.label as string;
```

Comment:
It looks like `item.props.label` has the type `OptionLabel`. Do we need to handle the case if the label is not a string? Looks like it could potentially be a Typography component from the [TypographyText](https://github.com/Khan/wonder-blocks/blob/4215976f666d1af4446334b7723a43d54cbcb085/packages/wonder-blocks-cell/src/util/types.ts#L36-L41) type!